sections/semantics-forms.include

```diff
@@ -1209,7 +1209,7 @@
   <pre highlight="html">
 &lt;form&gt;
   &lt;table&gt;
-    &lt;caption&gt;Example &lt;<code>label</code>:gt;'s <code>for</code> attribute with &lt;<code>label</code>&gt;&lt;/caption&gt;
+    &lt;caption&gt;Example &lt;label:gt;'s for attribute with &lt;label&gt;&lt;/caption&gt;
```

Comment:
There was a typo on that line as well as the ```<code>``` tags... So the closing angle backet for the 1st label currently says ```:gt;``` and this should be replaced with ```&gt;``` (just before ***'s***)